gobblin-yarn/src/main/java/gobblin/yarn/GobblinApplicationMaster.java

```diff
@@ -202,17 +166,12 @@ public synchronized void stop() {
 
     // Send a shutdown request to the containers as a second guard in case Yarn could not stop the containers
     sendShutdownRequest();
-
     try {
-      // Give the services 5 minutes to stop to ensure that we are responsive to shutdown requests
-      this.serviceManager.stopAsync().awaitStopped(5, TimeUnit.MINUTES);
-    } catch (TimeoutException te) {
-      LOGGER.error("Timeout in stopping the service manager", te);
+      super.stop();
```

Comment:
Doesn't this mean that `GobblinApplicationMaster` will only wait 5 seconds for shutdown now, rather than 5 minutes?